Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -622,6 +623,10 @@ def SwapKeepAndDrop(l):
         'keep *_*_GEMStripDigiSimLink_*',
         'keep *_*_ME0DigiSimLink_*',
         'keep *_*_ME0StripDigiSimLink_*'])
+
+phase2_ecal_devel.toModify(FEVTDEBUGHLTEventContent,
+    outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
+        'keep *_ecal*_*_*'])
```

Comment:
How big of the eventcontent we are adding here?

Reply:
I have not checked the size. This is a catch all mainly to keep all ECAL reco products, in particular the uncalibrated RecHits. But they are only kept if the `phase2_ecal_devel` modifier is given so all other WFs except 28234.61 should see no change.

Reply:
can modifications in this file be avoided?
Individual detectors or producer categories should be defined already in their corresponding areas and simply included in this file.
I see that this is already not followed from the cases above, but it should not be a case to pollute this file.